license-exceptions.json

```diff
@@ -39,6 +39,9 @@
     "sha.js@*": {
       "reason": "Licensed under MIT; https://github.com/crypto-browserify/sha.js#license-mit"
     },
+    "type-fest@*": {
+      "reason": "Dual-licensed under MIT or CC0-1.0; https://github.com/sindresorhus/type-fest#license"
+    },
```

Comment:
Exception needed to get `apache2-license-checker` to pass; this entire file needs reviewing/removing. Have spoke to @benjaminhobbs about this.

Reply:
@jamesdonoh It'd be useful to attach an issue explaining the reason for removing these exceptions, with this draft PR: https://github.com/bbc/simorgh/pull/1559, otherwise it'll be easy to lose track of doing this work.